src/server/game/Accounts/AccountMgr.cpp

```diff
@@ -390,6 +390,20 @@ std::string AccountMgr::CalculateShaPassHash(std::string const& name, std::strin
     return ByteArrayToHexStr(sha.GetDigest(), sha.GetLength());
 }
 
+bool AccountMgr::IsBannedAccount(std::string const& name)
+{
+    PreparedStatement* stmt = LoginDatabase.GetPreparedStatement(LOGIN_SEL_ACCOUNT_BANNED_BY_USERNAME);
+    stmt->setString(0, name);
+    PreparedQueryResult result = LoginDatabase.Query(stmt);
+
+    if (!result)
```

Comment:
`if (!result)
    return false;`

This is more clean to read and is more according to current coding etiquette

Reply:
Yes, the curly brackets below are deemed unnecessary when only one line is present.